src/main/java/edu/cornell/kfs/rass/RassConstants.java

```diff
@@ -38,5 +38,8 @@ public static boolean isSuccessfulResult(RassObjectUpdateResultCode resultCode)
     public static final String RASS_DEFAULT_PROPOSAL_AWARD_TYPE_PARAMETER = "RASS_DEFAULT_PROPOSAL_AWARD_TYPE";
     
     public static final String PROPOSAL_ORG_CHART = "IT";
+    
+    public static final String REPORTING_NAME = "reportingName";
+    public static final String REPORTS_TO_AGENCY_NUMBER = "reportsToAgencyNumber";
```

Comment:
Both of these constants already appear to be available in the KFSPropertyConstants class, so you don't need to add them here.